src/main/kotlin/andrasferenczi/configuration/ConfigurationData.kt

```diff
@@ -4,14 +4,15 @@ import andrasferenczi.templater.TemplateConstants
 
 // Input
 // Default values
-data class ConfigurationData(
+data class ConfigurationData constructor(
     val copyWithMethodName: String,
     val useRequiredAnnotation: Boolean,
     val useNewKeyword: Boolean,
```

Comment:
not related to null safety, but I think we can also change `useNewKeyword` to `false` by default (it's not needed anymore)